scripts/kibana_proxy.py

```diff
@@ -53,46 +53,51 @@ class KibanaProxy:
 
     def __init__(self, options) -> None:
         self.options = options
+        self.docker = docker.from_env()
+
+    def create_container(self, image: str, command=None, **kwargs):
+        image_name, tag = image.split(':')
+        self.docker.images.pull(image_name, tag=tag)
+        return self.docker.containers.create(image, command, **kwargs)
```

Comment:
I chose this form over

```suggestion
    try:
        return self.docker.containers.create(image, command, **kwargs)
    except docker.images.NotFoundError:
        self.docker.images.pull(image_name, tag=tag)
        return self.docker.containers.create(image, command, **kwargs)
```

for brevity. Is this the correct approach?

Reply:
Personally i like to be informed that something is being downloaded, if there is a slow connection and the images are taking a while to download, it would be nice to have an alert on in the logs, rather than me thinking "why is this taking a long time"